### PR TITLE
DBZ-818 Adjusting scale of BigDecimal values to column's scale if needed

### PR DESCRIPTION
**Pull request by gunnarmorling**
https://issues.jboss.org/browse/DBZ-818

### CODE REVIEW

File: debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleValueConverters.java [hunk 1]
```diff
@@ -235,21 +235,26 @@ protected Object convertDecimal(Column column, Field fieldDefn, Object data) {
...
-    @Override
-    protected Object convertNumeric(Column column, Field fieldDefn, Object data) {
-        if (data instanceof NUMBER) {
-            try {
-                data = ((NUMBER)data).bigDecimalValue();
-            }
-            catch (SQLException e) {
-                throw new RuntimeException("Couldn't convert value for column " + column.name(), e);
-            }
+    private BigDecimal withScaleAdjustedIfNeeded(Column column, BigDecimal data) {
```
Comment (jpechane) [inline debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleValueConverters.java]: The same code is present in postgres connector - maybe this is something that is common for all connectors https://github.com/debezium/debezium/blob/master/debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresValueConverter.java#L346

**Comment by gunnarmorling** [comment]
Yes, it's true. I wasn't quite sure though about the impact of changing it
globally, so I thought I'd stay on the safe side and apply it just here.
Can you open a JIRA issue for it? Thanks!

**Comment by jpechane** [comment]
https://issues.jboss.org/browse/DBZ-825

**Comment by jpechane** [comment]
@gunnarmorling Applied, thanks!

**Pull request merged**